### PR TITLE
Fix graph refresh fetching custom attributes

### PR DESCRIPTION
**Pull request by borod108**
Graph refresh fixed to fetch the custom attributes for both full and targeted refresh.

Fix: https://bugzilla.redhat.com/show_bug.cgi?id=1546792

### CODE REVIEW

File: app/models/manageiq/providers/redhat/inventory_collection_default/infra_manager.rb [hunk 1]
```diff
@@ -131,6 +131,23 @@ def root_folders(extra_attributes = {})
       attributes.merge!(extra_attributes)
     end
 
+    def custom_attributes(extra_attributes = {})
```
Comment (pkliczewski) [inline app/models/manageiq/providers/redhat/inventory_collection_default/infra_manager.rb]: Why not to update [here](https://github.com/ManageIQ/manageiq/blob/master/app/models/manager_refresh/inventory_collection_default/infra_manager.rb#L137) instead.

File: spec/vcr_cassettes/manageiq/providers/redhat/infra_manager/refresh/ovirt_sdk_refresh_recording.yml [hunk 2]
```diff
@@ -5253,6 +5253,12 @@ https://localhost:8443/ovirt-engine/api/vms/3a9401a0-bf3d-4496-8acf-edd3e903511f
                     </topology>
                 </cpu>
                 <cpu_shares>0</cpu_shares>
+            <custom_properties>
```
Comment (pkliczewski) [inline spec/vcr_cassettes/manageiq/providers/redhat/infra_manager/refresh/ovirt_sdk_refresh_recording.yml]: Please align it with existing content

File: spec/vcr_cassettes/manageiq/providers/redhat/infra_manager/refresh/ovirt_sdk_refresh_recording.yml [hunk 3]
```diff
@@ -5380,6 +5386,13 @@ https://localhost:8443/ovirt-engine/api/vms/3a9401a0-bf3d-4496-8acf-edd3e903511f
...
```
Comment (pkliczewski) [inline spec/vcr_cassettes/manageiq/providers/redhat/infra_manager/refresh/ovirt_sdk_refresh_recording.yml]: As above

File: app/models/manageiq/providers/redhat/inventory_collection_default/infra_manager.rb (hunk 1 above)
Comment (borod108) [reply app/models/manageiq/providers/redhat/inventory_collection_default/infra_manager.rb]: I am not sure what else the core part is used for, I will ask @ladas + its faster to merge.

File: spec/vcr_cassettes/manageiq/providers/redhat/infra_manager/refresh/ovirt_sdk_refresh_recording.yml (hunk 2 above)
Comment (borod108) [reply spec/vcr_cassettes/manageiq/providers/redhat/infra_manager/refresh/ovirt_sdk_refresh_recording.yml]: sure, actually it does not work for some reason, I will probably delete this part.

File: app/models/manageiq/providers/redhat/inventory_collection_default/infra_manager.rb [hunk 4]
```diff
@@ -131,6 +131,23 @@ def root_folders(extra_attributes = {})
       attributes.merge!(extra_attributes)
     end
 
+    def vm_and_template_ems_custom_fields(extra_attributes = {})
```
Comment (pkliczewski) [inline app/models/manageiq/providers/redhat/inventory_collection_default/infra_manager.rb]: @agrare do you think we should update here or move this change to core repo?

**Comment by borod108** [comment]
[Investigate bug Ladas found with targeted template, add to BZ if real](https://trello.com/c/wYDPDJ94/296-investigate-bug-ladas-found-with-targeted-template-add-to-bz-if-real)

**Comment by borod108** [comment]
@masayag please review

**Comment by borod108** [comment]
@ladas please review

**Comment by mwperina** [comment]
@simaishi Satoe, could you please backport to 5.9?

**Comment by masayag** [comment]
@miq-bot add_label bug

**Comment by simaishi** [comment]
Gaprindashvili backport details:

```
$ git log -1
commit 95115c13cf152a038af1ae366ef31ac1db112530
Author: Moti Asayag <masayag@redhat.com>
Date:   Thu Mar 1 12:52:57 2018 +0200

    Merge pull request #211 from borod108/cust_attrs
    
    Fix graph refresh fetching custom attributes
    (cherry picked from commit 8f9f2252f7881bf577b29efda25334526c6f70a4)
    
    Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1559624
```

**Pull request merged**